### PR TITLE
Add pointer for .NET debugging for Rider

### PR DESCRIPTION
**Pull request by briandealwis**
**Description**
`skaffold debug`'s setup for .NET Core does not work for JetBrains Rider.  Add a pointer to Cloud Code for IntelliJ's workaround.  The PR also puts the OmniSharp details into a separate call out.

### CODE REVIEW

File: docs/content/en/docs/workflows/debug.md [hunk 1]
```diff
@@ -272,6 +276,12 @@ your base image.  (`//` comments must be stripped.)
     }
 }
 ```
+{{< /alert >}}
+
+{{< alert title="JetBrains Rider" >}}
+This set up does not yet work with JetBrains' Rider,
```
Comment (etanshaul) [inline docs/content/en/docs/workflows/debug.md:282]: does not yet work [automatically | out of the box] with JetBrains Rider ?

Comment (etanshaul) [reply docs/content/en/docs/workflows/debug.md]: as discussed on chat:

> does not yet work automatically with Cloud Code for IntelliJ in JetBrains Rider

is even clearer

File: docs/content/en/docs/workflows/debug.md [hunk 2]
```diff
@@ -237,7 +239,14 @@ a debug launcher that examines the app command-line.
 
 Furthermore, your app must be built with the `--configuration Debug` options to disable optimizations.
 
-**Note for users of [VS Code's debug adapter for C#](https://github.com/OmniSharp/omnisharp-vscode):**
+
+{{< alert title="JetBrains Rider" >}}
+This set up does not yet work automatically with Cloud Code for IntelliJ in JetBrains Rider,
+but [there is a workaround](https://github.com/GoogleCloudPlatform/cloud-code-intellij/wiki/Manual-set-up-for-remote-debugging-in-Rider).
```
Comment (etanshaul) [inline docs/content/en/docs/workflows/debug.md:245]: nit: but maybe "there is a **manual** workaround"? to emphasis the distinction between a possible future automated solution

**Comment by etanshaul** [comment]
had a tiny suggestion. but LGTM.

**Review by tejal29 — APPROVED** [review]

**Pull request merged**